bullyBot.py

```diff
@@ -0,0 +1,47 @@
+#bullyBot.py
```

Comment:
I think this was supposed to be a shebang.  Shebangs help command lines know what interpreter to use without having to be explicit.  Google python shebang.

Reply:
Added shebang

Reply:
+1